src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/ruleNode.tsx

```diff
@@ -161,21 +191,81 @@ class RuleNode extends React.Component<Props> {
     );
   }
 
+  conditionallyRenderHelpfulBanner() {
+    const {data, project, organization} = this.props;
+    /**
+     * Would prefer to check if data is of `IssueAlertRuleAction` type, however we can't do typechecking at runtime as
+     * user defined types are erased through transpilation.
+     * Instead, we apply duck typing semantics here.
+     * See: https://stackoverflow.com/questions/51528780/typescript-check-typeof-against-custom-type
+     */
+    if (!data?.targetType) {
+      return null;
+    }
+
+    switch (data.targetType) {
+      case MailActionTargetType.IssueOwners:
+        return (
+          // Using boolean attributes on styled components cases a warning
+          // https://github.com/styled-components/styled-components/issues/1198
+          // Preferring to use boolean attributes to respect TS typing.
+          <Alert thinner type="warning">
+            {tct(
+              'If there are no matching [issueOwners], ownership is determined by the setting on [thisPage].',
+              {
+                issueOwners: (
+                  <ExternalLink href="https://docs.sentry.io/workflow/issue-owners/">
+                    {t('issue owners')}
+                  </ExternalLink>
+                ),
+                thisPage: (
+                  <ExternalLink
+                    href={`/settings/${organization.slug}/projects/${project.slug}/ownership/`}
+                  >
+                    {t('this page')}
```

Comment:
We should give a more useful string here

Reply:
@billyvg any suggestions?

Reply:
If there are no matching issue owners, ownership is determined by the [ownership settings]?

Reply:
sounds good. I might leave out the "the"

Reply:
Thanks, done!